pycloudlib/ibm_classic/cloud.py

```diff
@@ -0,0 +1,500 @@
+# This file is part of pycloudlib. See LICENSE file for license information.
+"""IBM Cloud type."""
+
+from typing import List, Literal, Optional, Tuple
+
+import SoftLayer  # type: ignore
+
+from pycloudlib.cloud import BaseCloud
+from pycloudlib.config import ConfigFile
+from pycloudlib.ibm_classic.errors import IBMClassicException
+from pycloudlib.ibm_classic.instance import IBMClassicInstance
+from pycloudlib.instance import BaseInstance
+
+
+class IBMClassic(BaseCloud):
+    """IBM Classic Class."""
+
+    _type = "ibmclassic"
+
+    def __init__(
+        self,
+        tag: str,
+        timestamp_suffix: bool = True,
+        config_file: Optional[ConfigFile] = None,
+        *,
+        sl_username: Optional[str] = None,
+        sl_api_key: Optional[str] = None,
+    ):
+        """Initialize base cloud class.
+
+        Args:
+            tag: string used to name and tag resources with
+            timestamp_suffix: Append a timestamped suffix to the tag string.
+            config_file: path to pycloudlib configuration file
+            sl_username: IBM Softlayer username
```

Comment:
I guess we could yeah. I'm slightly hesitant to because I don't want people trying to use their normal username (the softlayer username is entirely different) and also just wanted to distinguish the softlayer api key from the vpc api key. But I can agree to removing the softlayer naming, I will just want to add a section to the documentation explaining how to find these credentials (which i should really do anyway)